module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -652,6 +652,14 @@ public function getLanguages()
         return (array)($this->fields['language'] ?? []);
     }
 
+    /**
```

Comment:
We've customized this file in the TAMU modules directory. I think adding this change to the file [here](https://github.com/TAMULib/vufind/blob/tamu-vufind-9.0.3/module/TAMU/src/TAMU/RecordDriver/DefaultRecord.php) instead should get the same results and keep future upgrades simpler.

Reply:
It currently doesn't work when placed in our TAMU module directory, but you're right that it would make updates easier. I'll investigate what needs to be done to make the _local_ version work.